bootstrap/stage0/runtime/lib.h

```diff
@@ -280,42 +280,12 @@ struct ExplicitValueOrControlFlow {
         if constexpr (IsVoid<Value>)
             return;
         else
-            return move(value).template get<ExplicitValue<Value>>().value;
+            return move(move(value).template get<ExplicitValue<Value>>().value);
```

Comment:
I don't understand this `move()` you've added.

Reply:
I haven't added that move. I haven't updated bootstrap manually, I just copied `runtime/lib.h` over.

Reply:
Anyway, I've removed that change since it's irrelevant to the PR.